cluster-autoscaler/cloudprovider/azure/azure_util_test.go

```diff
@@ -21,8 +21,9 @@ import (
 	"net/http"
 	"testing"
 
-	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-12-01/compute"
```

Comment:
Can we do the azure sdk updates in a separate PR - just to keep this diff small and manageable?